packages/ember-simple-auth/tests/unit/session-stores/shared/storage-event-handler-behavior.js

```diff
@@ -52,8 +52,8 @@ export default function(options) {
         triggered = true;
       });
 
-      store.persist({ key: 'value' }); // this data will be read again when the event is handled so that no change will be detected
-      window.dispatchEvent(new StorageEvent('storage', { key: store.get('key') }));
+      store.persist({ someOtherKey: 'value' }); // this data will be read again when the event is handled so that no change will be detected
+      window.dispatchEvent(new StorageEvent('storage', { someOtherKey: store.get('key') }));
```

Comment:
Changed the keys because it seemed like there are some race conditions with other async tests.

Reply:
Hm, I'd suggest looking into this again – we shouldn't have race conditions in the test suite.